components/categories-filter/CategoryButton.jsx

```diff
@@ -7,14 +7,16 @@ export default function FilterBtn({
   buttonClick,
   category,
   isDisabled,
+  className,
 }) {
   return (
-    <li className="filter__item">
+    <li className={`filter__item ${className}`}>
```

Comment:
anti-pattern: _`li` as a root element_. Can we change it now?

Reply:
https://iamakulov.com/notes/react-antipattern-dont-make-li-your-components-root/

Reply:
Category Buttons aren't intended to use outside of filter. But I've changed them.

Reply:
thanks!